app/workers/articles/resave_for_association_worker.rb

```diff
@@ -0,0 +1,23 @@
+module Articles
+  class ResaveForAssociationWorker
```

Comment:
the name change [will cause queued jobs to fail](https://stackoverflow.com/questions/43526075/is-it-safe-to-rename-a-sidekiq-worker-with-jobs-still-in-the-queue). It's best to alias it somewhere similar to
```
Users::ResaveArticlesWorker = Articles::ResaveForAssociationWorker
```
should be safe to do that at the bottom of this file.

Reply:
actually what I mentioned above won't work because the args requirement changed. I think it's best to keep the old `app/workers/users/resave_articles_worker.rb` around for a bit then remove it.

Reply:
Thanks for the context.  Will do.  I'll refactor accordingly!

Reply:
@maestromac I pushed up some changes